Swiften/Elements/RosterItemPayload.h

```diff
@@ -41,12 +41,16 @@ namespace Swift {
                 unknownContent_ += c;
             }
 
+            void setChannel(bool isChannel) { isChannel_ = isChannel; }
+            bool getChannel() const { return isChannel_; }
```

Comment:
We usually name bools like this isX, rather than getX. Also, does it make sense to explicitly call it a MIXChannel, rather than Channel, to avoid ambiguity with other channels?